Mage.Sets/src/mage/cards/z/ZeriamGoldenWind.java

```diff
@@ -0,0 +1,44 @@
+package mage.cards.z;
+
+import java.util.UUID;
+import mage.MageInt;
+import mage.abilities.common.DealsCombatDamageToAPlayerTriggeredAbility;
+import mage.abilities.effects.common.CreateTokenEffect;
+import mage.constants.SubType;
+import mage.constants.SuperType;
+import mage.abilities.keyword.FlyingAbility;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.game.permanent.token.GriffinToken;
+
+/**
+ *
+ * @author freaisdead
+ */
+public final class ZeriamGoldenWind extends CardImpl {
+
+    public ZeriamGoldenWind(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.CREATURE}, "{3}{W}");
+        
+        this.addSuperType(SuperType.LEGENDARY);
+        this.subtype.add(SubType.GRIFFIN);
+        this.power = new MageInt(3);
+        this.toughness = new MageInt(4);
+
+        // Flying
+        this.addAbility(FlyingAbility.getInstance());
+
+        // Whenever a Griffin you control deals combat damage to a player, create a 2/2 white Griffin creature token with flying.
+        this.addAbility(new DealsCombatDamageToAPlayerTriggeredAbility(new CreateTokenEffect(new GriffinToken(), 1), true));
```

Comment:
This will only trigger when this creature deals combat damage, you need to use `DealsDamageToAPlayerAllTriggeredAbility` instead. Take a look at Boggard Mob.

Reply:
Oh okey, what's the difference?

Reply:
The one you have is "Whenever this creature deals combat damage" and the one you need is "Whenever a Griffin you control deals combat damage"

Reply:
I seeeee thanks